server/http.go

```diff
@@ -64,13 +64,7 @@ func (p *Plugin) connectUserToZoom(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	channelID := r.URL.Query().Get("channelID")
-	if channelID == "" {
-		http.Error(w, "channelID missing", http.StatusBadRequest)
-		return
-	}
-
-	state, err := p.storeUserState(userID, channelID)
```

Comment:
It looks like we don't actually have to include `channelID` in the user state we save in the OAuth flow here.